.changelog/226.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+Add API for metrics from an alternate filesystem root, allow OperatingSystem() to read from an alternate filesystem root
```

Comment:
The `OperatingSystem` part isn't needed anymore.

```suggestion
The sysinfo APIs (e.g. `Host()`, `Process()`) now accept an optional argument to force reading from an alternative filesystem root. This can be useful inside of containers to read data from the Linux host.
```